### ISSUE TITLE
Output Format

### ISSUE THREAD

**Issue by Chay16** [sshaoshuai/PointRCNN#27]
Hi sshoashui,
Thaanks you for releasing your code and the pretrained model. I'm a bit confuse when trying to plot the 3D predicted bounding boxes. I'm guessing that the output format (the first line of the prediction for 000001.txt) encodes the bounding box as follows:

Car -1 -1 -1.7723 753.6551 163.8756 814.0421 204.0525 **1.5462 (_height_) 1.6426 (_width_) 3.9845 (_length_) 7.0502 (_x_) 1.2010 (_y_) 29.7814 (_z_) -1.5398 (_theta_)** 1.6346

with y representing the elevation and the XZ plane representing the ground plane.

Am i right ? 
Regards,

**Comment by sshaoshuai** [comment]
You are right. Here the output form is consistent with the KITTI label files (KITTI rectified camera coordinate ). Please refer to the introduction of KITTI training labels for details.

**Comment by Chay16** [comment]
OK thanks for your answer ! Another point, is there a way to disable the calibration step easily to train PointRCNN on other annotated point clouds ?

**Comment by sshaoshuai** [comment]
You need to check the codes manually to update the codes where the calibrations are used. Most of them are in the dataset related codes, maybe you should write a new dataloader for your own point clouds.

**Comment by Chay16** [comment]
I will write a new dataloader, it will be more appropriate I think. Another point, why are you subsampling the point cloud into 16,384 points as mentioned in the paper? Is it only for a memory point of view ?

**Comment by sshaoshuai** [comment]
The PointNet++ needs a fixed number of input points and then I subsample 16384 points as the input.

**Comment by uranuscc** [comment]
this post has helped to clarify my doubts too. thanks.
